package.json

```diff
@@ -201,7 +201,7 @@
     "taskr": "1.1.0",
     "tree-kill": "1.2.2",
     "tsec": "0.2.1",
-    "turbo": "1.3.2-canary.1",
+    "turbo": "1.5.3",
```

Comment:
Upgrade of turbo relates to https://github.com/vercel/next.js/pull/40833#issuecomment-1256950320 but it wasn't the cause after all. Still doesn't hurt to upgrade 👍